cloud/extensions.rst

```diff
@@ -0,0 +1,68 @@
+.. _cloud_extensions:
+
+Extensions
+==========
+
+To keep a standard Cloud installation for all customers and improve our ability to troubleshoot and provide support, we do not provide superuser access to Cloud clusters. Thus customers are not able to install PostgreSQL extensions themselves.
+
+Generally there is no need to install extensions, however, because every Cloud cluster comes pre-loaded with many useful ones:
+
++--------------------+---------+------------+--------------------------------------------------------------------+
+|        Name        | Version |   Schema   |                            Description                             |
++====================+=========+============+====================================================================+
+| btree_gin          | 1.0     | public     | support for indexing common datatypes in GIN                       |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| btree_gist         | 1.2     | public     | support for indexing common datatypes in GiST                      |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| citext             | 1.3     | public     | data type for case-insensitive character strings                   |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| citus              | 7.0-15  | pg_catalog | Citus distributed database                                         |
```

Comment:
We may want to remove the version column because it could be a maintenance burden in the docs.